bspump/avro/loader.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+import json
+import fastavro
+import logging
+
+
+
+###
+
+L = logging.getLogger(__name__)
+
+###
+
+
+def load_avro_schema(config):
+    schema = config.get('schema')
+    if schema == '':
+        schema_file = config.get('schema_file')
+        if schema_file != '':
+            with open(schema_file, 'r') as fi:
+                schema = json.load(fi)
+
+    if schema == '':
+        raise RuntimeError("Avro scheme not read")
```

Comment:
The text of error is incorrect, it should be e.g. "AVRO schema is not configured."